lib/system_test/src/read_only_path.cpp

```diff
@@ -27,11 +27,13 @@ namespace metashell
 {
   namespace system_test
   {
-#ifdef _WIN32
     namespace
     {
+#if defined(_WIN32) || defined(__APPLE__)
```

Comment:
Why is this needed on `__APPLE__`?

Reply:
Thanks for noticing, this was left in accidentally